ledger-bridge.js

```diff
@@ -91,6 +110,12 @@ export default class LedgerBridge {
                     this.app = new LedgerEth(this.transport)
                 }
             } else if (this.transportType === 'webhid') {
```

Comment:
This handles a pre-existing bug, but one that was only noticeable with the changes made in this PR